setup.py

```diff
@@ -42,9 +42,11 @@
             + "webviz_subsurface.plugins:ParameterParallelCoordinates",
             "RunningTimeAnalysisFMU = webviz_subsurface.plugins:RunningTimeAnalysisFMU",
             "RelativePermeability = webviz_subsurface.plugins:RelativePermeability",
+            "RftPlotter =  webviz_subsurface.plugins:RftPlotter",
         ]
     },
     install_requires=[
+        "colorlover>=0.3.0",
```

Comment:
Is it possible to use https://plotly.com/python/builtin-colorscales/ instead of adding `colorlover` as dependency?

Reply:
Current `colorlover` interpolation:
`hsl(210.0, 100.0%, 98.0%)`
`hsl(208.0, 60.0%, 91.0%)`
`hsl(209.0, 56.0%, 85.0%)`
`hsl(200.0, 52.0%, 75.0%)`
`hsl(202.0, 56.0%, 62.0%)`

`f`-string, with endpoints `Blues[0]` and `Blues[5]`:
`hsl(210.0, 100.0%, 98.0%)`
`hsl(208.0, 89.0%, 89.0%)`
`hsl(206.0, 78.0%, 80.0%)`
`hsl(204.0, 67.0%, 71.0%)`
`hsl(202.0, 56.0%, 62.0%)`

`f`-string*, with endpoints `Blues[0]` and `Blues[6]`:
`hsl(210.0, 100.0%, 98.0%)`
`hsl(208.25, 88.25%, 86.25%)`
`hsl(206.5, 76.5%, 74.5%)`
`hsl(204.75, 64.75%, 62.75%)`
`hsl(203.0, 53.0%, 51.0%)`

`f`-string, with endpoints `Blues[0]` and `Blues[-1]`:
`hsl(210.0, 100.0%, 98.0%)`
`hsl(210.75, 97.25%, 81.0%)`
`hsl(211.5, 94.5%, 64.0%)`
`hsl(212.25, 91.75%, 47.0%)`
`hsl(213.0, 89.0%, 30.0%)`

\* Corresponding `f`-string:
```python
[f"hsl({210*(1-x) + 203*x}, {100*(1-x) + 53*x}%, {98*(1-x)+51*x}%)" for x in np.linspace(0, 1, 5)]
```